ingest-app/src/cmr/ingest/api/generic_documents.clj

```diff
@@ -171,10 +171,16 @@
      (:body (tgen/update-generic request-context [provider-id native-id] metadata-json)))))
 
 (defn delete-generic-document
-  "TODO: Generic work: add delete"
-  [request]
-  (println "stub function: delete ")
-  {:status 501})
+  "Deletes a generic document in elastic search and creates a tombstone in the database. Returns a 201
```

Comment:
Tiny nitpick, elasticsearch is a single word

Reply:
fixed.